pkg/apis/extensions/register.go

```diff
@@ -63,8 +64,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ReplicaSetList{},
 		&PodSecurityPolicy{},
 		&PodSecurityPolicyList{},
-		&NetworkPolicy{},
-		&NetworkPolicyList{},
+		&networking.NetworkPolicy{},
```

Comment:
Could you confirm my understanding of the impact of this PR? I think this PR doesn't affect users at all. Users will still be able to access extensions/v1beta1/networkpolicy and networking.k8s.io/v1/networkpolicy. And the object can be stored in both versions in the etcd, according to user's choice.